system-parachains/asset-hubs/asset-hub-kusama/src/lib.rs

```diff
@@ -112,7 +112,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("statemine"),
 	impl_name: create_runtime_str!("statemine"),
 	authoring_version: 1,
-	spec_version: 1_002_006,
+	spec_version: 1_002_008,
```

Comment:
if we massage the dependency bumps a bit we might get away without having to bump the AH spec versions, right @claravanstaden ?

I wouldn't release new AH runtime versions that bring nothing new

Reply:
actually no, I'm wrong.

Even if this PR does not change anything for AHs, there have been other things merged in the meantime relevant to AHs.

So all good with bumping this.

Reply:
@acatangiu thanks! I see CI is complaining about the Kusama relay version too: https://github.com/polkadot-fellows/runtimes/actions/runs/9740656775/job/26878232348?pr=365

I bumped both the Polkadot and Kusama relay versions: https://github.com/polkadot-fellows/runtimes/pull/365/commits/b2a0b268db849763f1bee19a2ff52a02d0e14fba

Is that OK?

Reply:
yes, all good 👍 we have changelog entries for them too I see